commit 14a5ee8ce4fbc764bfd89a8c2401d4cc3a363c5c (refs/changes/98/3198/1)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-01-11T21:38:08+00:00 (1 year, 9 months ago)
    
    MANTA-3285 muskie coal example config refers to "authservice.coal.joyent.us"

diff --git a/etc/config.coal.json b/etc/config.coal.json
index ea3521f..494db73 100644
--- a/etc/config.coal.json
+++ b/etc/config.coal.json
@@ -45,7 +45,7 @@
     "cueballHttpAgent": {
         "resolvers": ["nameservice.coal.joyent.us"],
         "initialDomains": [
-          "authservice.coal.joyent.us"
+          "authcache.coal.joyent.us"
         ],
         "ping": "/ping",
         "pingInterval": 60000,
